llvm/lib/IR/Core.cpp

```diff
@@ -2921,6 +2921,17 @@ int LLVMGetSignednessInfo(LLVMValueRef Inst) {
   }
 }
 
+/*-- Get the variable in the llvm.dbg.declare instructions .................--*/
```

Comment:
Maybe we need to add a note in the documentation to indicate that this function is a customized version of SBIP?

Something like this: 

https://github.com/sbip-sg/llvm-project/blob/ec52fbac6b8f148a9663224cc4e20f3482a13701/llvm/lib/Analysis/Analysis.cpp#L164-L165